### PR TITLE
fix(deps) Upgrade puppeteer to v 19.5.2

### PR DESCRIPTION
**Pull request by aitchiss**
Closes https://github.com/netlify/netlify-plugin-lighthouse/issues/510

Originally, [the upgrade to Puppeteer 19.x was failing](https://github.com/netlify/netlify-plugin-lighthouse/pull/499), and their docs advised `createBrowserFetcher` was to be deprecated, which is why we paused renovate updating this dependency.

Now, however, it seems the issue we were seeing with a change to the args for `createBrowserFetcher` has been resolved, and their docs have been updated to state `createBrowserFetcher` is [deprecated only for users of `puppeteer-core`](https://pptr.dev/api/puppeteer.puppeteernode.createbrowserfetcher/) (not us)

TL;DR a minor version bump on their side has already fixed the issues, and we no longer have a deprecation to worry about 🎉 

Cypress specs are passing, and I've tested by pointing a site's `@netlify/plugin-lighthouse` at this branch, and all looks fine to me.

### CODE REVIEW

**Review by jackbrewer — APPROVED** [review]
🎉
